src/data/list/big_operators.lean

```diff
@@ -61,6 +61,11 @@ lemma prod_eq_pow_card (l : list M) (m : M) (h : ∀ (x ∈ l), x = m) :
   l.prod = m ^ l.length :=
 by rw [← prod_repeat, ← list.eq_repeat.mpr ⟨rfl, h⟩]
 
+@[to_additive]
+lemma prod_eq_one {l : list M} (hl : ∀ (x ∈ l), x = (1 : M)) :
+  l.prod = 1 :=
+trans (prod_eq_pow_card l 1 hl) (one_pow l.length)
```

Comment:
I guess this matches `finset.prod_eq_one`. Can you mention this in the PR description? Do we have `multiset.prod_eq_one` already?

Reply:
I added that as well. Also added comments for the fact that we have `iff` versions of these statements for `canonically_ordered_monoid`, but these apply more generally to just a `monoid`